src/schemes/boundary/system.jl

```diff
@@ -398,3 +398,29 @@ end
 function calculate_dt(v_ode, u_ode, cfl_number, system::BoundarySystem)
     return Inf
 end
+
+function initialize!(system::BoundarySPHSystem, neighborhood_search)
+    initialize_colorfield!(system, system.boundary_model, neighborhood_search)
+    return system
+end
+
+function initialize_colorfield!(system, boundary_model, neighborhood_search)
+    return system
+end
+
+function initialize_colorfield!(system, ::BoundaryModelDummyParticles, neighborhood_search)
+    system_coords = system.coordinates
+    (; smoothing_kernel, smoothing_length) = system.boundary_model
+
+    foreach_point_neighbor(system, system,
+                           system_coords, system_coords,
+                           neighborhood_search,
+                           points=eachparticle(system)) do particle, neighbor, pos_diff,
+                                                           distance
+        system.boundary_model.cache.colorfield_bnd[particle] += kernel(smoothing_kernel,
+                                                                       distance,
+                                                                       smoothing_length)
+        system.boundary_model.cache.neighbor_count[particle] += 1
+    end
+    return system
+end
```

Comment:
Please add a test.

Reply:
I have added a test but this test will change in all subsequent PRs

Reply:
I think that is better than reducing coverage here and potentially forgetting about it.